exporter/shadow-cljs.edn

```diff
@@ -12,7 +12,7 @@
               :after-load app.core/start}
 
    :compiler-options
-   {:output-feature-set :es8
+   {:output-feature-set :es2020
```

Comment:
I think it would be cleaner correct this out of this MR

Reply:
True. I'm going to do this in a separate commit, direct to develop.